envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotUpdaterTest.kt

```diff
@@ -51,6 +51,7 @@ import java.util.concurrent.Executors
 import java.util.concurrent.TimeUnit
 import java.util.function.Consumer
 
+@Suppress("LargeClass")
```

Comment:
Maybe we need to think about breaking it up?

Reply:
In my opinion we can have large test classes

Reply:
Large test files suggests that test subject contain too much logic.

Reply:
We agreed to refactor it in other PR